src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -62,16 +68,23 @@
 public class SkroutzGreekStemmer {
   public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
   private final CharArraySet stopwords;
+  private Map<String,char[]> step_0_exceptions;
 
   public SkroutzGreekStemmer(final CharArraySet stopwords) {
     this.stopwords = stopwords;
+    this.step_0_exceptions = new HashMap(0);
```

Comment:
We are using this in case step 0 exception weren't loaded. The init size 0 is just for extra efficiency, to avoid the default size.